bee-api/bee-rest-api/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "bee-rest-api"
+version = "0.1.0"
+authors = ["IOTA Stiftung"]
+edition = "2018"
+description = "A REST API implementation for IOTA node software."
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = ["iota", "tangle", "bee", "framework", "api"]
```

Comment:
I think I'ld remove "tangle", and add the keyword "node" just before "api"

Reply:
I think if one has to be removed to put node, it should be framework. These are keywords to help people find our crates and if they know about iota, there is a big chance they look for tangle. Idk.